components/splash/cards/forDevelopersCards.js

```diff
@@ -53,7 +57,7 @@ const ForDevelopersCards = (props) => (
     {...props}
   >
     {cardData.map((card, i) => (
-      <Card key={i} icon={card.icon} heading={card.heading} text={card.text} />
+      <Card key={i} icon={card.icon} heading={card.heading} text={card.text} showHowItWorksLink={i === 0} />
```

Comment:
put whether or not to show how it works into the card object instead of relying on idx